src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -486,6 +486,7 @@ let private resourceSummary =
      T               0   0
      Depth           0   0
      Width           1   1
+     QubitCount      1   1
```

Comment:
This would necessarily be the same as Width in this case. Do you see any case where this would be different? 
If they are the same, could you please remove it to avoid unnecessary confusion? I.e. just looking at this, people might expect this to be the total number of qubits use (without reuse), which it isn't.

Reply:
Yes it may be different. When we optimize for Depth, QubitCount will be smaller than Width. Also QubitCount does count borrowed qubits as extra qubits, but Width does not, as they are the same underlying qubits.

Reply:
@DmitryVasilevsky That borrowed qubits are included in QubitCount is important to document. Thanks!